phase1/gce/do

```diff
@@ -17,15 +24,16 @@ generate_token() {
 gen() {
   TOKEN=$(generate_token)
   mkdir -p ${TMP_DIR}
-  jsonnet -J ../../ --multi ${TMP_DIR} all.jsonnet
+  jsonnet -J ../../ --multi ${TMP_DIR} --tla-code-file cfg=${CONFIG_JSON_FILE} all.jsonnet
   echo "kubeadm_token = \"$TOKEN\"" > ${CLUSTER_NAME}/terraform.tfvars
 }
 
 deploy() {
   gen
   terraform apply -var-file=${CLUSTER_NAME}/terraform.tfvars -state=${CLUSTER_NAME}/terraform.tfstate ${TMP_DIR}
 
-  if [[ "${WAIT_FOR_KUBECONFIG:-}" == "y" ]]; then
+  PHASE2=$(jq -r '.phase2.provider' ${CONFIG_JSON_FILE})
```

Comment:
I'm on the fence about this. When I originally introduced `WAIT_FOR_KUBECONFIG`, it was to very conservatively allow some (hypothetical) customers to set up security-conscious clusters where ssh wasn't necessarily enabled, or port 22 could be firewalled off, or they just didn't want to automatically pull their administrative credentials back to the client as part of cluster deploy. This flexibility might have been unnecessarily conservative, and maybe we don't actually have users who would want to turn the functionality off and still use kubeadm.

CC @kubernetes/kubernetes-anywhere-maintainers for more opinions about this change.